scripts/entrypoint.sh

```
@@ -0,0 +1,66 @@
+#!/bin/sh
+set -e
+
+echo "[Entrypoint] Starting volume seeding process..."
+
+# Helper function for seeding
+# $1: Pretty name for logs (e.g., "Logos")
+# $2: Source directory in image (e.g., "/app/.initial-logos")
+# $3: Target volume directory in container (e.g., "/app/data/images/logos")
+seed_volume() {
+  local PNAME="$1"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)